src/main/java/invtweaks/InvTweaksItemTree.java

```diff
@@ -4,13 +4,20 @@
 import invtweaks.api.IItemTree;
 import invtweaks.api.IItemTreeCategory;
 import invtweaks.api.IItemTreeItem;
+import net.minecraft.creativetab.CreativeTabs;
 import net.minecraft.item.Item;
+import net.minecraft.item.ItemArmor;
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.JsonToNBT;
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.nbt.NBTUtil;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.ResourceLocation;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
 import net.minecraftforge.oredict.OreDictionary;
+import scala.Int;
```

Comment:
I think this is the wrong import for that. There isn't much reason for anything from `scala` to be imported.

(Added later: Also looks unused)

Reply:
I had been using Int.MaxValue().  I've been swapping that out and forgot to remove the import.